std_msgs/Header header
erau_fusion_msgs/tracked_veh_object[32] objects
